foundation-extension/4-projects/business_unit_1/development/boa_secret.tf

```diff
@@ -55,7 +55,8 @@ locals {
       "roles/compute.securityAdmin"
     ],
     cicd = [
-      "roles/binaryauthorization.attestorsViewer"
+      "roles/binaryauthorization.attestorsViewer",
+      "roles/resourcemanager.projectIamAdmin"
```

Comment:
Why do we need to give this IAM admin?

Reply:
4-projects Custom TF SA added permission Project IAM Admin on cicd prj so Bastion SA permission can be added (Bug)
else this will fail https://github.com/GoogleCloudPlatform/terraform-example-foundation-app/blob/main/5-infrastructure/modules/bastion/main.tf#L60